### PR TITLE
Added some new CS faculty @ Uni Goettingen

### PR DESCRIPTION
**Pull request by aecker**
Added/modified the following people: Alexander S. Ecker (edit), Fabian H. Sinz, Constantin Pape, Bela Gipp (edit), Anne-Christin Hauschild, Niels Grabe, Bernhard Schmitzer, Tim Beißbarth, Burkhard Morgenstern, Dagmar Krefting, Ulrich Sax

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

### CODE REVIEW

**Comment by aecker** [comment]
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

File: csrankings-a.csv [hunk 1]
```diff
@@ -1691,6 +1691,7 @@ Anne Roudaut,University of Bristol,http://big.cs.bris.ac.uk/people/roudaut,03epP
 Anne S. Hsu,Queen Mary University of London,http://www.eecs.qmul.ac.uk/~ahsu,2dCb2U0AAAAJ
 Anne Villems,University of Tartu,http://www.ut.ee/en/anne-villems,E3la6C8AAAAJ
 Anne-Cécile Caron,CRIStAL,http://cristal.univ-lille.fr/~caronc,NOSCHOLARPAGE
+Anne-Christin Hauschild,University of Göttingen,https://achauschild.com,Khifj_MAAAAJ
```
Comment (emeryberger) [inline csrankings-a.csv:1733]: Two affiliations given on home page. Please clarify.

File: csrankings-b.csv [hunk 2]
```diff
@@ -861,6 +862,7 @@ Burkay Genç,Hacettepe University,http://burkay.tk,w-pGWs4AAAAJ
 Burkhard C. Wünsche,University of Auckland,https://www.cs.auckland.ac.nz/~burkhard,pgBZIHYAAAAJ
 Burkhard Claus Wünsche,University of Auckland,https://www.cs.auckland.ac.nz/~burkhard,pgBZIHYAAAAJ
 Burkhard Freitag,University of Passau,http://www.im.uni-passau.de,NOSCHOLARPAGE
+Burkhard Morgenstern,University of Göttingen,http://gobics.de,IiGchZoAAAAJ
```
Comment (emeryberger) [inline csrankings-b.csv:879]: Not in CS. Please clarify inclusion eligibility.

File: csrankings-n.csv [hunk 3]
```diff
@@ -413,6 +413,7 @@ Nicu Sebe,University of Trento,http://disi.unitn.it/~sebe,tNtjSewAAAAJ
 Niculae Sebe,University of Trento,http://disi.unitn.it/~sebe,tNtjSewAAAAJ
 Niels A. Taatgen,University of Groningen,http://www.ai.rug.nl/~niels,F5FI6uYAAAAJ
 Niels Anne Taatgen,University of Groningen,http://www.ai.rug.nl/~niels,F5FI6uYAAAAJ
+Niels Grabe,University of Göttingen,http://tigacenter.bioquant.uni-heidelberg.de/grabe.niels.html,zkw0tYgAAAAJ
```
Comment (emeryberger) [inline csrankings-n.csv:424]: Listed at Heidelberg and not obviously in CS, either. Please clarify.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by emeryberger** [comment]
Bump @aecker

File: csrankings-a.csv (hunk 1 above)
Comment (aecker) [reply csrankings-a.csv:1733]: Recently moved to Uni Göttingen Medical Center Department of Medical Informatics. Marburg is old affiliation and phasing out. See CV: https://achauschild.com/curriculum-vitae/

File: csrankings-b.csv (hunk 2 above)
Comment (aecker) [reply csrankings-b.csv:879]: Research in bioinformatics, contributes to computer science bachelor's and master's degrees and is part of the CS PhD program. See PhD program website: https://www.uni-goettingen.de/en/138991.html#PP

File: csrankings-n.csv (hunk 3 above)
Comment (aecker) [reply csrankings-n.csv:424]: Recently moved to Uni Göttingen Medical Center; website not updated yet. Primary affiliation is Göttingen. Professorship for Data Analysis and Fusion for Imaging. Will be affiliated with CS PhD program as well. Official university announcement here: https://www.uni-goettingen.de/en/1.+januar+2022+bis+31.+märz+2022/663802.html (in German)

**Comment by aecker** [comment]
I hope this clarifies these three cases. Let me know if you need additional information or want us to remove some of them.

**Comment by aecker** [comment]
I believe it's ready to merge now, @emeryberger ?

**Pull request merged**